certbot/cli.py

```diff
@@ -709,6 +709,11 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         help="With the register verb, indicates that details associated "
              "with an existing registration, such as the e-mail address, "
              "should be updated, rather than registering a new account.")
+    helpful.add(
+        "register", "--deactivate", action="store_true",
+        help="Irrevocably deactivate your account. Certificates associated "
+             "with your account can still be revoked, but they can not be "
+             "renewed.")
```

Comment:
While Boulder doesn't do this, in the ACME spec it says:
```
   Once an account is deactivated... It is up to server policy...
   whether to revoke certificates issued by that account...
```
I think this potential side effect should be mentioned here.

I'm also not sure about the phrase "but they can not be renewed". From a Certbot perspective, if the user runs `certbot certonly` once to associate the lineages with a new account, they can still renew their certs. I think either this phrase should be removed or changed to something like "but they can not be renewed without first obtaining certificates from a new account".